### PR TITLE
Change 'subcmd' to 'command' and command group in tests and code

### PR DESCRIPTION
**Pull request by KSchopmeyer**
We have decided on the names "command" and "command group" to define the
components of the pywbemcli cmd line.  However they were all named
subcmds in the tests, test methods, test definition tools. Changed all of that to command and command_group just to be consistent.

The word subcmd should not occur any more.

This just changes the text and the key names in the test dictionaries. 

It does not change code.

Also changed the test file names from test_*_subcmd.py to test_*_cmd.py

### CODE REVIEW

File: pywbemtools/pywbemcli/_cmd_server.py [hunk 1]
```diff
@@ -375,7 +375,7 @@ def cmd_server_centralinsts(context, options):
                     scoping_class=options['scoping_class'],
                     scoping_path=options['scoping_path'],
                     reference_direction=options['reference_direction'])
-                row.append(":".join([str(p) for p in ci]))
+                row.append("/n".join([str(p) for p in ci]))
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_server.py]: This change fixes issue #363. Either say that in commit message title and description, or move to a separate PR.

Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_server.py]: My BooBoo.  It was not supposed to.  I accidently leaked one pr into another. I am taking this one out and making it separate.

Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_server.py]: REMOVED.
